cunumeric/config.py

```diff
@@ -268,44 +437,44 @@ class CuNumericTunable(IntEnum):
 class _FFTType:
     def __init__(
         self,
-        name,
-        type_id,
-        input_dtype,
-        output_dtype,
-        single_precision,
-        complex_type=None,
-    ):
+        name: str,
+        type_id: int,
+        input_dtype: npt.DTypeLike,
+        output_dtype: npt.DTypeLike,
```

Comment:
I'm not about this location specifically, but I  just wanted to note that there were definitely places that `DTypeLike` was required rather than `np.dtype[Any]`. I think dtype handling and nomenclature is one of the ares that could benefit from a project-wide review at some point.